styles.css

```diff
@@ -0,0 +1,100 @@
+:root {
+  --projectPurple: #6070ff;
+  --headerColor: #172b4d;
+}
+
+* {
+  text-decoration: none;
+  margin: 0;
+  width: 100%;
+  height: 100px;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+}
+
+.logo {
+  font-family: Poppins, sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+  color: var(--projectPurple);
+}
+
+.hamburguer {
+  color: var(--projectPurple);
+}
+
+.introduction {
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  margin-right: 50px;
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.name {
+  padding-top: 70px;
+  color: var(--headerColor);
+  font-family: Poppins, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: left;
+  margin-left: 20px;
+}
+
+.header-banner {
+  width: 100%;
+  height: calc(100vh - 100px);
+  background-image: url("./images/background.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.connect-text {
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  margin-top: 10px;
+  color: var(--projectPurple);
+  margin-left: 20px;
+}
+
+.header-flex {
+  display: flex;
+  flex-direction: column;
+}
+
+.social-media-icons {
+  display: flex;
+  column-gap: 15px;
+  margin-left: 15px;
+  margin-top: 5px;
+  align-items: center;
+}
+
+.social-media-icons img {
+  align-self: center;
+}
+
+.linkedin-logo {
+  height: 30px;
+  width: auto;
+}
```

Comment:
- [ ] [REQUIRED] Please give a style like the required design because the placement and style of the elements in your design are not like [Figma's design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1).

![image](https://user-images.githubusercontent.com/63915024/185475145-c84748de-17f0-4aba-92bb-8cdc7c4c5213.png)